src/helpers/localStorageHelpers.js

```diff
@@ -81,17 +81,16 @@ export const removeFavoriteCategory = (category) => {
   return true;
 };
 
-export const getDrafts = () => {
-  return store.get('drafts') || {};
-};
+export const getLayout = () =>
+  store.get('layout') || 'card';
```

Comment:
For nested path you need to implement slicer function. 

https://github.com/elgerlambert/redux-localstorage/blob/master/README.md#configslicer